src/unix/bsd/netbsdlike/netbsd/riscv64.rs

```diff
@@ -0,0 +1,24 @@
+use PT_FIRSTMACH;
+
+pub type c_long = i64;
+pub type c_ulong = u64;
+pub type c_char = u8;
+pub type __cpu_simple_lock_nv_t = ::c_int;
+
+// should be pub(crate), but that requires Rust 1.18.0
```

Comment:
You can use cfgs if it should be gated, see https://github.com/rust-lang/libc/blob/main/build.rs

Reply:
Hm, I'll admit this is copy-pasted from the origin.  That said, this particular comment and construct seems to be prevalent in both freebsd and openbsd target descriptions, and the other netbsd descriptions as well.
And ... I'll admit that it's not self-evident to me how *exactly* to do what the comment implies.

Reply:
Ah, I see! Could you remove them then?

Reply:
I removed the comment from the riscv64.rs description.
Do you want me to remove it from all the other NetBSD descriptions as part of this pull request?

Reply:
@JohnTitor Maybe I don't quite understand what you are asking.  Can the tidy-up of the conditional in the various netbsd *.rs files please be postponed to a separate pull request?  And can this particular pull request please be applied?  libc is vendored "like crazy", and this is hurting portability for this particular platform.